src/program/UplcProgramV1.js

```diff
@@ -173,14 +174,19 @@ export class UplcProgramV1 {
     /**
      * @param {Option<UplcValue[]>} args - if None, eval the root term without any applications, if empy: apply a force to the root term
      * @param {number[]} costModelParams
+     * @param {Option<UplcLoggingI>} logOptions?
```

Comment:
As mentioned above: it might make sense to move both optional args into a separate `options` object

Reply:
If it's required in the CekMachine, we could make the same change here.  On the other hand, if there's an important use-case for silent logging without the mental load of a no-op `{logPrint()}{}}`, then an `options` object could work well :+1: